validation_pipeline.ipynb

```diff
@@ -25,12 +25,14 @@
         "DEBUG = False #@param {type:\"boolean\"}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    mean_isoscapes = [
&nbsp; &nbsp; means_isoscape,
</pre><p>This is kind of counterintuitive for me. Can we clarify that those are <em>oxygen</em> means somehow?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Yeah, renamed the variable.